doc-content/enterprise-only/getting-started-kogito/ref-kogito-microservices-app-examples.adoc

```diff
@@ -17,4 +17,19 @@ Decision services::
 * `dmn-drools-quarkus-metrics` and `dmn-drools-springboot-metrics`: A decision service (on {QUARKUS} or Spring Boot) that enables and consumes the runtime metrics monitoring feature in {KOGITO}.
 * `pmml-quarkus-example` and `pmml-springboot-example`: A decision service (on {QUARKUS} or Spring Boot) that uses PMML.
 
-For more information about using DMN, DRL, and PMML models in {PRODUCT}, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_], {URL_DEVELOPING_DECISION_SERVICES}#assembly-drl-rules[_{DRL_RULES}_], and {URL_DEVELOPING_DECISION_SERVICES}#assembly-pmml-models[_{PMML_MODELS}_].
+Process services::
+* `process-scripts-quarkus` and `process-scripts-springboot`: A process service (on {QUARKUS} and Spring Boot) that invokes scripts within a process.
+* `process-service-calls-quarkus` and `process-service-calls-springboot`: A process service (on {QUARKUS} and Spring Boot) that invokes service calls within a process.
+* `process-rest-service-call-quarkus` and `process-rest-service-call-springboot`:  A process service (on {QUARKUS} and Spring Boot) that invokes REST service calls within a process.
```

Comment:
```suggestion
* `process-rest-service-call-quarkus` and `process-rest-service-call-springboot`:  A process service (on {QUARKUS} and Spring Boot) that invokes REST service calls within a process using a service task.
```

Reply:
This was added to the wrong examples, one line above. It should be at `process-rest-service-call-*` examples.